aiographite/aiographitesend.py

```diff
@@ -0,0 +1,447 @@
+import asyncio
+from graphite_escaping import metrics_name_to_graphite, metrics_name_from_graphite
+import pickle
+import struct
+import socket
+import os
+import time
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+SUPPORT_PROTOCOLS = ["plaintext", "pickle"]
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class AsyncioGraphiteSendService(object):
+
+	def __init__(self, graphite_server, graphite_port, protocol = "pickle"):
+		"""
+			Must call __init__ before use
```

Comment:
I think you probably don't need this comment. it's impossible to call a method without instantiating the object first.